pkg/kubectl/cmd/apiresources/apiversions.go

```diff
@@ -56,19 +56,22 @@ func NewCmdApiVersions(f cmdutil.Factory, ioStreams genericclioptions.IOStreams)
 		Long:    "Print the supported API versions on the server, in the form of \"group/version\"",
 		Example: apiversionsExample,
 		Run: func(cmd *cobra.Command, args []string) {
-			cmdutil.CheckErr(o.Complete(f))
+			cmdutil.CheckErr(o.Complete(f, cmd, args))
 			cmdutil.CheckErr(o.RunApiVersions())
 		},
 	}
 	return cmd
 }
 
-func (o *ApiVersionsOptions) Complete(f cmdutil.Factory) error {
+func (o *ApiVersionsOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []string) error {
 	var err error
 	o.discoveryClient, err = f.ToDiscoveryClient()
 	if err != nil {
 		return err
 	}
+	if len(args) != 0 {
```

Comment:
This is usually the first check in `Complete`. Additionally all error should start with lower case letter.